### PR TITLE
benchmarks/R-cran-rbenchmark:1.0.0

### PR DESCRIPTION
**Pull request by yzgyyang**
```
new port: benchmarks/R-cran-rbenchmark: Benchmarking routine for R

Approved by: lwhsu
```

### CODE REVIEW

File: patches/benchmarks.R-cran-rbenchmark.1.0.0.diff [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     SUBDIR += apib
+     SUBDIR += autobench
+diff --git a/benchmarks/R-cran-rbenchmark/Makefile b/benchmarks/R-cran-rbenchmark/Makefile
+new file mode 100644
+index 000000000000..b3d6890b5401
+--- /dev/null
++++ b/benchmarks/R-cran-rbenchmark/Makefile
+@@ -0,0 +1,16 @@
++# $FreeBSD$
++# Created by: Guangyuan Yang <ygy@FreeBSD.org>
```
Comment (lwhsu) [inline patches/benchmarks.R-cran-rbenchmark.1.0.0.diff:20]: I found that those two lines are usually in reverse order, but this is definitely not important :-)
(especially we will drop `$FreeBSD$` when switching to git.

File: patches/benchmarks.R-cran-rbenchmark.1.0.0.diff [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
++rbenchmark is inspired by the Perl module Benchmark, and is intended to
++facilitate benchmarking of arbitrary R code. The library consists of just one
++function, benchmark, which is a simple wrapper around system.time. Given a
++specification of the benchmarking process (counts of replications, evaluation
++environment) and an arbitrary number of expressions, benchmark evaluates each
++of the expressions in the specified environment, replicating the evaluation as
++many times as specified, and returning the results conveniently wrapped into a
++data frame.
++
++WWW: http://rbenchmark.googlecode.com
```
Comment (lwhsu) [inline patches/benchmarks.R-cran-rbenchmark.1.0.0.diff:59]: This link gives 404, is this project still alive?

**Review by lwhsu — CHANGES_REQUESTED** [review]

File: patches/benchmarks.R-cran-rbenchmark.1.0.0.diff (hunk 1 above)
Comment (yzgyyang) [reply patches/benchmarks.R-cran-rbenchmark.1.0.0.diff:20]: Oh nice find! Will switch all these in this and all other patches.

File: patches/benchmarks.R-cran-rbenchmark.1.0.0.diff (hunk 2 above)
Comment (yzgyyang) [reply patches/benchmarks.R-cran-rbenchmark.1.0.0.diff:59]: It's a nice find! The module still works on my machine, though the code repo is not updated for more than 5 years. In this case, I shall not add it to ports, and I need a reminder to check the project status before bringing things in :)

**Comment by yzgyyang** [comment]
Closing since the project is not active anymore, pointed out by @lwhsu. Also a mental note to self that I should check more closely about the project status before creating ports.

**Pull request closed without merging**